include/mbedtls/ssl.h

```diff
@@ -1956,6 +1965,27 @@ void mbedtls_ssl_conf_authmode( mbedtls_ssl_config *conf, int authmode );
 */
 void mbedtls_ssl_tls13_conf_early_data( mbedtls_ssl_config *conf,
                                         int early_data_enabled );
+#endif /* MBEDTLS_SSL_CLI_C */
+
+#if defined(MBEDTLS_SSL_SRV_C)
+/**
+ * \brief Set the max_early_data_size parameter.
+ *
+ * \param[in] conf                  The SSL configuration to use.
+ * \param[in] max_early_data_size   The maximum amount of 0-RTT data.
+ *                                  - 0 Disable 0-RTT feature.
+ *
+ * \note    max_early_data_size MUST be smaller than
+ *          MBEDTLS_SSL_MAX_EARLY_DATA_SIZE. Otherwise,
+ *          MBEDTLS_SSL_MAX_EARLY_DATA_SIZE will be used.
+ *
+ * \warning This interface is experimental and may change without notice.
+ *
+ */
+void mbedtls_ssl_tls13_conf_max_early_data_size(
```

Comment:
Seems we can combine this function with mbedtls_ssl_tls13_conf_early_data(), it's easy. But I am not sure whether we should leave it one independent API.

Reply:
I prefer a separate  API. 
As my understand, your meaning is `mbedtls_ssl_tls13_conf_early_data( conf, enable, max_early_data_size )` .
Compare with separate API, I think it might confuse end user.

Reply:
Per offline discussion, It looks @xkqian and me has some difference. And @xkqian does not very insistent. I prefer keep it. 
@ronald-cron-arm what's your opinion?

Reply:
I am rather in favor of two APIs `mbedtls_ssl_tls13_conf_early_data()` and `mbedtls_ssl_tls13_conf_max_early_data_size()` as it stands. Mainly because otherwise `mbedtls_ssl_tls13_conf_early_data()` would be different on client and server side or have a dummy parameter (max_early_data_size) on client side.

Reply:
One question about the callback in server side. Should we add one more api to configure it? or we have no need of callback functions?

Reply:
It is invalid topic now. According to the design, no callback needed.